Makefile.include

```diff
@@ -172,7 +172,9 @@ endif
 ifeq ($(MAKE_NET),MAKE_NET_IPV6)
   CFLAGS += -DNETSTACK_CONF_WITH_IPV6=1
   MODULES += os/net/ipv6
-else
+endif
+
+ifeq ($(MAKE_NET),MAKE_NET_NULLNET)
   CFLAGS += -DNETSTACK_CONF_WITH_NULLNET=1
   MODULES += os/net/nullnet
```

Comment:
I suggest `MODULES` is moved up to the other `MAKE_NET_NULLNET` check at line 170 and that this second `MAKE_NET_NULLNET` check is simply removed.

Reply:
Oh yes good catch! Will do